share/spice/drupal/drupal.js

```diff
@@ -0,0 +1,78 @@
+(function (env) {
+    "use strict";
+    env.ddg_spice_drupal = function(api_result){
+
+        // Validate the response
+        if (!api_result || api_result.list === "") {
+            return Spice.failed('drupal');
+        }
+
+        // Render the response
+        Spice.add({
+            id: "drupal",
+            name: "Software",
+            data: api_result.list[0],
+            meta: {
+                sourceName: "Drupal.org",
+                sourceUrl: 'https://www.drupal.org/project/' + api_result.list[0].field_project_machine_name 
+            },
+            normalize: function(item) {
+                var boxData = [{heading: 'Project Information:'}];
+
+                if (item.field_project_homepage.url) {
```

Comment:
field_project_homepage is an empty array if left blank. This is what happens when I try to check if item.field_project_homepage exists:

![image](https://cloud.githubusercontent.com/assets/607807/8458796/023ed25e-1fe7-11e5-8065-212a1124f887.png)

I can probably try ```if (item.field_project_homepage !== [])```